### PR TITLE
Using commons

### PR DESCRIPTION
**Pull request by Cerfoglg**

### CODE REVIEW

File: dbms/sqlquery/mysql/mysqldump/.project [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>benchflow-collector-mysqldump</name>
+	<name>benchflow-collector-dump</name>
```
Comment (VincenzoFerme) [inline dbms/sqlquery/mysql/mysqldump/.project]: Why did you change it? Does it work for all the databases?

File: dbms/sqlquery/mysql/mysqldump/src/cloud/benchflow/collectors/Dump.go [hunk 2]
```diff
@@ -5,50 +5,12 @@ import (
     "net/http"
     "os"
     "os/exec"
-    "github.com/minio/minio-go"
-    "log"
+    //"github.com/minio/minio-go"
+    "github.com/Cerfoglg/commons/src/minio"
```
Comment (VincenzoFerme) [inline dbms/sqlquery/mysql/mysqldump/src/cloud/benchflow/collectors/Dump.go]: Should be referring to the following repo: https://github.com/benchflow/commons

File: environment/logs/src/cloud/benchflow/collectors/Logs.go [hunk 3]
```diff
@@ -4,11 +4,11 @@ import (
 	"bufio"
 	"fmt"
 	"github.com/fsouza/go-dockerclient"
-	"github.com/minio/minio-go"
+	//"github.com/minio/minio-go"
+	"github.com/Cerfoglg/commons/src/minio"
```
Comment (VincenzoFerme) [inline environment/logs/src/cloud/benchflow/collectors/Logs.go]: Should be referring to the following repo: https://github.com/benchflow/commons

File: environment/stats/src/cloud/benchflow/collectors/stats.go [hunk 4]
```diff
@@ -5,22 +5,21 @@ import (
...
 	"net/http"
 	"os"
-	"os/exec"
 	"strings"
-	"time"
 	"sync"
 	"github.com/fsouza/go-dockerclient"
-	"github.com/minio/minio-go"
+	//"github.com/minio/minio-go"
+	"github.com/Cerfoglg/commons/src/minio"
```
Comment (VincenzoFerme) [inline environment/stats/src/cloud/benchflow/collectors/stats.go]: Should be referring to the following repo: https://github.com/benchflow/commons

File: files/zip/src/cloud/benchflow/collectors/zip.go [hunk 5]
```diff
@@ -4,58 +4,17 @@ import (
     "fmt"
     "net/http"
     "os"
-    "os/exec"
     "strings"
-    "github.com/minio/minio-go"
-    "log"
+    "github.com/Cerfoglg/commons/src/minio"
```
Comment (VincenzoFerme) [inline files/zip/src/cloud/benchflow/collectors/zip.go]: Should be referring to the following repo: https://github.com/benchflow/commons

**Comment by VincenzoFerme** [comment]
@Cerfoglg Does this Pull Request solve #21?

**Comment by Cerfoglg** [comment]
@VincenzoFerme Should be fixed now, also fixes #21.
